contents/docs/session-replay/manual.mdx

```diff
@@ -17,95 +14,36 @@ Session replay enables you to record users navigating through your website and p
 Above is a recording of posthog.com… on posthog.com – very meta.
 </Caption>
 
-## How to enable session replay
+## Step 1: Install our JavaScript web library
 
-Session replay can only be used with our [JavaScript web library](/docs/integrate/client/js) and requires the feature to be enabled in your PostHog [Project Settings](https://app.posthog.com/project/settings).
+First, you need to install our JavaScript library. You can do so by either:
 
-Once enabled, the library will start recording sessions by default.
-
-<blockquote class="warning-note">
-    Session Replay does not work if you send data using Segment's SDK as this data is not collected. If you use
-    Segment, you may want to add the PostHog library too (make sure to only send regular event data from one source).
-</blockquote>
-
-It can be toggled on and off in the JS library by appropriately setting the `disable_session_recording` flag in the [config](/docs/integrate/client/js/#config). 
-
-Users who [opt out](/docs/libraries/js#opt-users-out) of event capturing will not have their sessions recorded.
-
-## How to watch recordings
-
-To watch recordings, you can visit the '[Replay](https://app.posthog.com/replay)' page. When watching replays, you can change the playback speed, as well as select the option 'skip inactivity' - this will skip parts of the recording where the user was inactive on the page.
-
-You can also watch recordings by clicking on any data point in an insight. This will open the list of persons and display a "watch recording" button if it is available. This is specially useful in [funnels](/docs/product-analytics/funnels), where you can drill down and watch recordings of users who converted or dropped off. 
-
-You can also click into the person to view all their recordings.
-
-![How to watch recordings via insights](../../images/docs/session-replay/how-to-watch-recordings.mp4)
-
-## How to ignore sensitive elements
-
-You may want to hide sensitive text or elements in your replays. See our [privacy controls](/docs/session-replay/privacy) docs for how to do this.
-
-## Console logs recording
-
-PostHog can also capture console logs from your application. This is useful for debugging and providing extra context on what is happening in your users' browser environment. 
-
-As console logs can contain sensitive information, we do not capture these logs automatically. You can enable this feature globally from your [project settings](https://app.posthog.com/project/settings) or client-side by setting `enable_recording_console_log: true` in our [JavaScript web library config](/docs/integrate/client/js/#config).
+import Snippet from "../../../integrate/\_snippets/install-js-snippet.mdx"
+import Badge from "../../../getting-started/\_snippets/badge.mdx"
+import InstallJSViaNPM from "../../../integrate/\_snippets/install-js-via-npm.mdx"
```

Comment:
Should we import these at the very top of the file?

Reply:
Perhaps. Its not necessary though. More of a styling issue